decisions/0003-integrate-with-the-jemalloc-allocator.md

```diff
@@ -0,0 +1,35 @@
+# 3. Integrate with the jemalloc allocator
+
+Date: 2022-03-12
+
+## Status
+
+Proposed
+
+## Context
+
+Memory pools are an important technique to improve application performance.
+The router already uses a custom memory pool (`src/alloc_pool.c`).
+Proton currently does not use any such mechanism.
```

Comment:
I should say that it does not really matter that much, because Dispatch takes care to avoid allocations in proton.